### PR TITLE
Break the rabbit URL into its component variables

### PR DESCRIPTION
**Pull request by nealedj**
## Motivation and Context
Currently the only way to set the RabbitMQ connection variables is via the compond URL parameters. This isn't ideal for when we'll want to be able to pull in the username and password from Kubernetes secrets.

## What has changed
Breaks out the Rabbit URL into its component parameters. 

## How to test?
`make test`

### CODE REVIEW

File: app/rabbit_helper.py
Comment (jcox-dev) [inline app/rabbit_helper.py]: `parameters` doesn't exist in `init_rabbitmq` parameter list?

File: app/rabbit_helper.py
Comment (jcox-dev) [inline app/rabbit_helper.py:74]: 👍

**Review by jcox-dev — CHANGES_REQUESTED** [review]
Unexpected param comment in one of the helper functions - otherwise looks useful!

NB. these changes will impact at least the following:
- [environment variables in docker-dev](https://github.com/ONSdigital/census-rm-docker-dev)
- [k8s deployments](https://github.com/ONSdigital/census-rm-kubernetes)

**Comment by nealedj** [comment]
Can be merged once these are merged:

- https://github.com/ONSdigital/census-rm-kubernetes/pull/45
- https://github.com/ONSdigital/census-rm-docker-dev/pull/15

**Review by jcox-dev — APPROVED** [review]

**Review by davidmort — APPROVED** [review]

**Pull request merged**
